### PR TITLE
Updated README.md with Adding new resources.

### PR DESCRIPTION
**Pull request by jakaya123**
I have included links to free ebooks that are super useful to open source contributors.

### CODE REVIEW

**Review by raisedadead — COMMENTED** [review]
Thanks for the PR.

I think while this could be useful. I think we should avoid adding resources which are not the direct authors/publishers of a book to avoid having to do the diligence on copyright etc.

The linked resource maybe following this, but the last thing we want it to be promoting it without diligence?

File: README.md [hunk 1]
```diff
@@ -125,6 +125,8 @@ Search links that point directly to suitable issues to contribute to on GitHub.
 
 ## Open Source books
 - [Producing Open Source Software](https://producingoss.com/) - Producing Open Source Software is a book about the human side of Open Source development. It describes how successful projects operate, the expectations of users and developers, and the culture of free software.
+- [Pro Git (Second Edition)](https://www.freetechbooks.com/pro-git-second-edition-t1202.html) - A fully-updated guide to Git and its usage in the modern world. This book teaches readers how to master the world of distributed version workflow, use the distributed features of Git to the full, and extend Git to meet their every need.
```
Comment (erictleung) [inline README.md:128]: This book is already freely available in the previous section on Using Version Control https://github.com/freeCodeCamp/how-to-contribute-to-open-source/blob/master/README.md#L117

```suggestion
```

So we can just remove this.

File: README.md [hunk 2]
```diff
@@ -125,6 +125,8 @@ Search links that point directly to suitable issues to contribute to on GitHub.
...
+- [The Architecture of Open Source Applications, Volume II: Structure, Scale, and a Few More Fearless Hacks](https://www.freetechbooks.com/the-architecture-of-open-source-applications-volume-ii-structure-scale-and-a-few-more-fearless-hacks-t1266.html) -The authors of twenty-four open source applications explain how their software is structured, and why. From web servers and compilers to health record management systems, they are covered here to help you become a better developer.
```
Comment (erictleung) [inline README.md:129]: Part of this is already in our list, but shown as just the git tool. We can update this link to point to all the books that are already freely available.

```suggestion
- [The Architecture of Open Source Applications](https://www.aosabook.org/en/index.html) - The authors of twenty-four open source applications explain how their software is structured, and why. From web servers and compilers to health record management systems, they are covered here to help you become a better developer.
```

**Review by erictleung — CHANGES_REQUESTED** [review]
Thanks for the changes. See comments for some feedback.

**Review by erictleung — APPROVED** [review]
LGTM! 👍

**Pull request merged**